docs/dev/table/sql.md

```diff
@@ -813,6 +813,222 @@ LIMIT 3
 
 {% top %}
 
+### Top-N
+
+TopN is used to calculate the maximum/minimum N records in a stream. It can be flexibly completed based on OVER window aggregation. The grammar is shown as below:
```

Comment:
`The grammar is shown as below:`? but below is sentence too? where is the grammar?

Reply:
Oh! Good catch! I missed to remove this line.